third_party_release

```diff
@@ -0,0 +1,7 @@
+benchmark=v1.5.3
+googletest=release-1.8.0-2523-ga6dfd3ac
+ms-gsl=v1.0.0-393-g6f45293
+nlohmann-json=v3.9.1
+opentelemetry-proto=v0.11.0
+prometheus-cpp=v0.11.0
+vcpkg=2020.04-2702-g5568f110b
```

Comment:
Sorry if I am missing something. What is the purpose of adding this file, and also adding an option in `do_ci.sh` to create this file? Is it meant to be a central place to list versions of all the otel-cpp dependencies?

Reply:
I believe it's a good idea to have the versions documented somewhere so the users can download the versions of the dependencies that a release has been published with. This would be obsolete if the plan is to not support older versions though.

Reply:
Ok, that makes sense. Just wondering if this should also contain the dependencies not coming from submodules, eg gRPC(v1.39.1), thrift (0.14.1), abseil(20210324.0)?

Reply:
yes absolutely. Just added them to the script.